content/download.html

```diff
@@ -113,34 +143,65 @@ <h3><img src="/images/platforms/appimage-small.svg" class="downloadsmallplatform
-  <pre>$ flatpak install --from https://flathub.org/repo/appstream/net.openra.OpenRA.flatpakref</pre></p>
-  <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
-  <p class="downloadthirdparty">The Flatpak sandbox interferes with the in-game mod switching feature between official and community mods.<br />We recommend that users who would like to use this feature run the AppImage releases instead.</p>
-  <h3><img src="/images/platforms/arch-small.svg" class="downloadsmallplatform" />Arch Linux</h3>
-  <p>Stable releases are available in the <a href="https://www.archlinux.org/packages/community/any/openra/">official Arch repository</a>, and can be installed using <span style="font-family: monospace;">pacman</span>:<pre>$ pacman -S openra </pre></p>
-  <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
-  <h3><img src="/images/platforms/gentoo-small.svg" class="downloadsmallplatform" />Gentoo</h3>
-  <p>Unofficial packages for both releases and playtests are available via the &ldquo;dr&rdquo; overlay:</p>
-  <pre>$ emerge -av layman
+  <h3>Community maintained packages</h3>
+  <p>Community maintained packages exist for a viariety of Linux distributions. We do not maintain the external package source, so there may be delays when a new version is released. Please contact the downstream repository maintainers about any packaging issues.</p>
+  <div class="distros">
+    <button class="accordion">
+      <img src="/images/platforms/flatpak-small.png" />
+      <span>Flatpak</span>
+    </button>
+    <span class="downloadthirdparty">An unofficial OpenRA Flatpak is available on <a href="https://flathub.org/apps/details/net.openra.OpenRA">Flathub</a>.</span>
+    <div class="panel">
+      <div class="codebox">
+        <pre>$ flatpak install --from https://flathub.org/repo/appstream/net.openra.OpenRA.flatpakref</pre>
+      </div>
+    </div>
+    <button class="accordion">
+      <img src="/images/platforms/arch-small.svg" />
+      <span>Arch Linux</span>
+    </button>
+    <span class="downloadthirdparty">Stable releases are available in the <a href="https://www.archlinux.org/packages/community/any/openra/">official Arch repository</a>.</span>
+    <div class="panel">
+      <div class="codebox">
+        <pre>$ pacman -S openra</pre>
+      </div>
+    </div>
+    <button class="accordion">
+      <img src="/images/platforms/gentoo-small.svg" />
+      <span>Gentoo</span>
+    </button>
+    <span class="downloadthirdparty">Unofficial packages for both releases and playtests are available via the <a href="https://github.com/cerebrum/dr">dr overaly</a>.</span>
```

Comment:
We've already said that all of these are unofficial, so these descriptions can be shortened e.g. "Release and playtest packages are available via ..."

Reply:
Done, can the Flatpak description stay like this?